CHANGELOG.md

```diff
@@ -1,8 +1,9 @@
 Change log
 -----------
 
+* Resinhup: fix update failure on Intel NUC due to partition matching [Gergely]
 * Plymouth: change splash image scaling to full screen instead of half [Gergely]
-* Resinhip: bump default resinhup container to v1.2.1 (Alpine Linux) [Gergely]
+* Resinhup: bump default resinhup container to v1.2.1 (Alpine Linux) [Gergely]
```

Comment:
This must be done now through versionist by using "Changelog-entry" and "Change-type" entries in one of the commits from your PRs